test/unit/mock/profiles/invalid-include-controls/README.md

```diff
@@ -0,0 +1,3 @@
+# Invalid Include Controls
+
+This profile contains an `include_controls` line with a invalid reference
```

Comment:
Personally, I prefer to delete the README (which is optional) and place the description of the test profile in the profile metadata (which is "less optional").

Reply:
I agree on both, been a while since I've created one of these.